components/sup/src/manager/service/spec.rs

```diff
@@ -225,6 +255,40 @@ impl serde::Serialize for ServiceBind {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub enum StartStyle {
+    Persistent,
+    Transient,
+}
+
+impl Default for StartStyle {
+    fn default() -> StartStyle {
+        StartStyle::Transient
```

Comment:
I'd say we want this to default to `StartStyle::Persistent` to favor service state sticking around. If a particular invocation is intended to be transient (`hab start` perhaps), we can explicitly feed this value to override a permanent default.